draft-private-access-tokens.md

```diff
@@ -451,6 +449,12 @@ key from the Issuer, and validate that the message matches the hash of the origi
 TokenChallenge for this session, SHA256(TokenChallenge), and that the version of the
 Token matches the version in the TokenChallenge.
 
+If a Client's issuance request fails with a 401 error, as described in {{request-two}},
+the Client MUST react to the challenge as if it could not produce a valid Authorization
+response.
```

Comment:
Normally, this would be a server workflow, would it not? Clients may not have a workflow here... perhaps the Client could respond with an empty response and allow the Server to follow the auth failure workflow as necessary?